2021-05-23 13:31:49,941 INFO    MainThread:1312 [internal.py:wandb_internal():88] W&B internal server running at pid: 1312, started at: 2021-05-23 13:31:49.933119
2021-05-23 13:31:49,943 INFO    WriterThread:1312 [datastore.py:open_for_write():77] open: c:\Users\cdami\Documents\Python Scripts\Spatiotemporal-Autoencoder-Anomaly-Detection\wandb\run-20210523_133148-t3wjp0k1\run-t3wjp0k1.wandb
2021-05-23 13:31:49,944 DEBUG   HandlerThread:1312 [handler.py:handle_request():121] handle_request: check_version
2021-05-23 13:31:49,952 DEBUG   SenderThread:1312 [sender.py:send():176] send: header
2021-05-23 13:31:49,952 DEBUG   SenderThread:1312 [sender.py:send_request():186] send_request: check_version
2021-05-23 13:31:50,125 DEBUG   SenderThread:1312 [sender.py:send():176] send: run
2021-05-23 13:31:50,497 DEBUG   HandlerThread:1312 [handler.py:handle_request():121] handle_request: run_start
2021-05-23 13:31:50,500 INFO    SenderThread:1312 [dir_watcher.py:__init__():168] watching files in: c:\Users\cdami\Documents\Python Scripts\Spatiotemporal-Autoencoder-Anomaly-Detection\wandb\run-20210523_133148-t3wjp0k1\files
2021-05-23 13:31:50,500 INFO    SenderThread:1312 [sender.py:_start_run_threads():696] run started: t3wjp0k1 with start time 1621769508
2021-05-23 13:31:50,500 DEBUG   SenderThread:1312 [sender.py:send():176] send: summary
2021-05-23 13:31:50,501 INFO    SenderThread:1312 [sender.py:_save_file():821] saving file wandb-summary.json with policy end
2021-05-23 13:31:50,511 DEBUG   HandlerThread:1312 [meta.py:__init__():39] meta init
2021-05-23 13:31:50,511 DEBUG   HandlerThread:1312 [meta.py:__init__():53] meta init done
2021-05-23 13:31:50,511 DEBUG   HandlerThread:1312 [meta.py:probe():210] probe
2021-05-23 13:31:50,521 DEBUG   HandlerThread:1312 [meta.py:_setup_git():200] setup git
2021-05-23 13:31:50,617 DEBUG   HandlerThread:1312 [meta.py:_setup_git():207] setup git done
2021-05-23 13:31:50,617 DEBUG   HandlerThread:1312 [meta.py:_save_pip():57] save pip
2021-05-23 13:31:50,617 DEBUG   HandlerThread:1312 [meta.py:_save_pip():71] save pip done
2021-05-23 13:31:50,618 DEBUG   HandlerThread:1312 [meta.py:_save_conda():78] save conda
2021-05-23 13:31:51,514 INFO    Thread-12 :1312 [dir_watcher.py:_on_file_created():216] file/dir created: c:\Users\cdami\Documents\Python Scripts\Spatiotemporal-Autoencoder-Anomaly-Detection\wandb\run-20210523_133148-t3wjp0k1\files\conda-environment.yaml
2021-05-23 13:31:51,514 INFO    Thread-12 :1312 [dir_watcher.py:_on_file_created():216] file/dir created: c:\Users\cdami\Documents\Python Scripts\Spatiotemporal-Autoencoder-Anomaly-Detection\wandb\run-20210523_133148-t3wjp0k1\files\requirements.txt
2021-05-23 13:31:51,514 INFO    Thread-12 :1312 [dir_watcher.py:_on_file_created():216] file/dir created: c:\Users\cdami\Documents\Python Scripts\Spatiotemporal-Autoencoder-Anomaly-Detection\wandb\run-20210523_133148-t3wjp0k1\files\wandb-summary.json
2021-05-23 13:31:58,456 DEBUG   HandlerThread:1312 [meta.py:_save_conda():86] save conda done
2021-05-23 13:31:58,457 DEBUG   HandlerThread:1312 [meta.py:probe():252] probe done
2021-05-23 13:31:58,551 DEBUG   SenderThread:1312 [sender.py:send():176] send: files
2021-05-23 13:31:58,551 INFO    SenderThread:1312 [sender.py:_save_file():821] saving file wandb-metadata.json with policy now
2021-05-23 13:31:58,579 INFO    Thread-12 :1312 [dir_watcher.py:_on_file_modified():229] file/dir modified: c:\Users\cdami\Documents\Python Scripts\Spatiotemporal-Autoencoder-Anomaly-Detection\wandb\run-20210523_133148-t3wjp0k1\files\conda-environment.yaml
2021-05-23 13:31:58,579 INFO    Thread-12 :1312 [dir_watcher.py:_on_file_created():216] file/dir created: c:\Users\cdami\Documents\Python Scripts\Spatiotemporal-Autoencoder-Anomaly-Detection\wandb\run-20210523_133148-t3wjp0k1\files\wandb-metadata.json
2021-05-23 13:31:59,169 INFO    Thread-17 :1312 [upload_job.py:push():133] Uploaded file C:\Users\cdami\AppData\Local\Temp\tmpxb79j6egwandb\21tjhbx9-wandb-metadata.json
2021-05-23 13:32:15,149 WARNING MainThread:1312 [internal.py:is_dead():366] Internal process exiting, parent pid 22836 disappeared
2021-05-23 13:32:15,149 ERROR   MainThread:1312 [internal.py:wandb_internal():143] Internal process shutdown.
2021-05-23 13:32:15,663 INFO    WriterThread:1312 [datastore.py:close():258] close: c:\Users\cdami\Documents\Python Scripts\Spatiotemporal-Autoencoder-Anomaly-Detection\wandb\run-20210523_133148-t3wjp0k1\run-t3wjp0k1.wandb
2021-05-23 13:32:15,663 INFO    HandlerThread:1312 [handler.py:finish():632] shutting down handler
2021-05-23 13:32:15,663 INFO    SenderThread:1312 [sender.py:finish():925] shutting down sender
2021-05-23 13:32:15,663 INFO    SenderThread:1312 [dir_watcher.py:finish():282] shutting down directory watcher
2021-05-23 13:32:15,693 INFO    SenderThread:1312 [dir_watcher.py:finish():312] scan: c:\Users\cdami\Documents\Python Scripts\Spatiotemporal-Autoencoder-Anomaly-Detection\wandb\run-20210523_133148-t3wjp0k1\files
2021-05-23 13:32:15,695 INFO    SenderThread:1312 [dir_watcher.py:finish():318] scan save: c:\Users\cdami\Documents\Python Scripts\Spatiotemporal-Autoencoder-Anomaly-Detection\wandb\run-20210523_133148-t3wjp0k1\files\conda-environment.yaml conda-environment.yaml
2021-05-23 13:32:15,695 INFO    SenderThread:1312 [dir_watcher.py:finish():318] scan save: c:\Users\cdami\Documents\Python Scripts\Spatiotemporal-Autoencoder-Anomaly-Detection\wandb\run-20210523_133148-t3wjp0k1\files\config.yaml config.yaml
2021-05-23 13:32:15,696 INFO    SenderThread:1312 [dir_watcher.py:finish():318] scan save: c:\Users\cdami\Documents\Python Scripts\Spatiotemporal-Autoencoder-Anomaly-Detection\wandb\run-20210523_133148-t3wjp0k1\files\requirements.txt requirements.txt
2021-05-23 13:32:15,698 INFO    SenderThread:1312 [dir_watcher.py:finish():318] scan save: c:\Users\cdami\Documents\Python Scripts\Spatiotemporal-Autoencoder-Anomaly-Detection\wandb\run-20210523_133148-t3wjp0k1\files\wandb-metadata.json wandb-metadata.json
2021-05-23 13:32:15,698 INFO    SenderThread:1312 [dir_watcher.py:finish():318] scan save: c:\Users\cdami\Documents\Python Scripts\Spatiotemporal-Autoencoder-Anomaly-Detection\wandb\run-20210523_133148-t3wjp0k1\files\wandb-summary.json wandb-summary.json
2021-05-23 13:32:15,702 INFO    SenderThread:1312 [file_pusher.py:finish():176] shutting down file pusher
2021-05-23 13:32:15,702 INFO    SenderThread:1312 [file_pusher.py:join():181] waiting for file pusher
2021-05-23 13:32:16,398 INFO    Thread-19 :1312 [upload_job.py:push():133] Uploaded file c:\Users\cdami\Documents\Python Scripts\Spatiotemporal-Autoencoder-Anomaly-Detection\wandb\run-20210523_133148-t3wjp0k1\files\config.yaml
2021-05-23 13:32:16,406 INFO    Thread-18 :1312 [upload_job.py:push():133] Uploaded file c:\Users\cdami\Documents\Python Scripts\Spatiotemporal-Autoencoder-Anomaly-Detection\wandb\run-20210523_133148-t3wjp0k1\files\conda-environment.yaml
2021-05-23 13:32:16,414 INFO    Thread-21 :1312 [upload_job.py:push():133] Uploaded file c:\Users\cdami\Documents\Python Scripts\Spatiotemporal-Autoencoder-Anomaly-Detection\wandb\run-20210523_133148-t3wjp0k1\files\wandb-summary.json
2021-05-23 13:32:17,463 INFO    Thread-20 :1312 [upload_job.py:push():133] Uploaded file c:\Users\cdami\Documents\Python Scripts\Spatiotemporal-Autoencoder-Anomaly-Detection\wandb\run-20210523_133148-t3wjp0k1\files\requirements.txt
2021-05-23 13:32:17,906 INFO    MainThread:1312 [internal.py:handle_exit():78] Internal process exited
